setup.py

```diff
@@ -38,6 +38,12 @@ def generate_long_description() -> str:
     return long_description
 
 
+extra_spark = ["pyspark"]
+extra_biology = ["biopython"]
+extra_chemistry = ["rdkit"]
+extra_engineering = ["unyt"]
+extra_all = extra_biology + extra_chemistry + extra_engineering + extra_spark
```

Comment:
I think `extra_chemistry` will cause it to fail if someone installs `pyjanitor[all]`. Could we comment it out until later?

Reply:
@ericmjl fixed :)